src/utilities/fetchAndExtractZip.utils.ts

```diff
@@ -22,7 +22,7 @@ const isDirectoryNode = memoize((node: FileNode | DirectoryNode): node is Direct
 
 const directoryCache = new Map<string, FileStructure>();
 
-const processZipContent = async (zip: JSZip): Promise<FileStructure> => {
+export const processZipContent = async (zip: JSZip): Promise<FileStructure> => {
```

Comment:
No need if we're using `fetchAndUnpackZip`

Reply:
reverted